packages/@aws-cdk/aws-stepfunctions-tasks/lib/eks/call.ts

```diff
@@ -0,0 +1,125 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Construct } from 'constructs';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * Properties for calling a EKS endpoint with EksCall
+ */
+export interface EksCallProps extends sfn.TaskStateBaseProps {
+
+  /**
+   * Name of the cluster
+   */
+  readonly clusterName: string;
+
+  /**
+   * Base 64 encoded certificate data required to communicate with your cluster
+   */
+  readonly certificateAuthority: string;
```

Comment:
this is available as `clusterCertificateAuthorityData` from an `ICluster`

Reply:
Will use EKS L2 Construct instead in next commit.